### PR TITLE
findContours return three values

### LINKED ISSUE #10: ValueError: too many values to unpack (expected 2)
When I try to `python3 main.py`, I've got this error, even with the default `input.png`
```
Executing phase: dress_to_correct
Executing phase: correct_to_mask
Executing phase: mask_to_maskref
Executing phase: maskref_to_maskdet
Executing phase: maskdet_to_maskfin
Traceback (most recent call last):
  File "main.py", line 30, in <module>
    main()
  File "main.py", line 21, in main
    watermark = process(dress)
  File "/content/deepnude_official/run.py", line 144, in process
    maskfin = create_maskfin(maskref, maskdet)
  File "/content/deepnude_official/opencv_transform/maskdet_to_maskfin.py", line 34, in create_maskfin
    bodypart_list = extractAnnotations(maskdet);
  File "/content/deepnude_official/opencv_transform/maskdet_to_maskfin.py", line 101, in extractAnnotations
    tits_list = findBodyPart(maskdet, "tit")
  File "/content/deepnude_official/opencv_transform/maskdet_to_maskfin.py", line 168, in findBodyPart
    contours, hierarchy = cv2.findContours(color_mask,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)
ValueError: too many values to unpack (expected 2)
```

### PR DESCRIPTION
**Pull request by jollheef**
fixes #10

### CODE REVIEW

**Comment by rom1504** [comment]
not needed, just use opencv 4

**Comment by harryyuanfeng** [comment]
how to use use opencv 4?

**Comment by rom1504** [comment]
pip install opencv-python==4.1.0.25

**Comment by rayryeng** [comment]
You can just do `contours, hierarchy = cv2.findContours(color_mask,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)[-2:]`.  This way it doesn't matter which version of OpenCV you use.  If you're using v3, it will always ignore the first element of the tuple and if you're using v4, it doesn't change anything.

File: opencv_transform/maskdet_to_maskfin.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def findBodyPart(image, part_name):
 		color_mask = cv2.inRange(image, f1, f2)
 
 	#find contours:
-	contours, hierarchy = cv2.findContours(color_mask,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)
+	_, contours, hierarchy = cv2.findContours(color_mask,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)
```
Comment (rayryeng) [inline opencv_transform/maskdet_to_maskfin.py]: Consider modifying this to `contours, hierarchy = cv2.findContours(color_mask,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)[-2:]`.  This way people don't have to upgrade to v4 if they don't want to and it works regardless of v3 or v4.

**Pull request closed without merging**